taskcall/manifest.json

```diff
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dd54da03-0a8c-4796-aaa6-61eeb04e611b",
+  "app_id": "taskcall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and centralize Datadog incidents with TaskCall.",
+    "title": "TaskCall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Incident Notifications",
+        "image_url": "images/incident_notifications.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Incident Details",
+        "image_url": "images/incident_details.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Impact Visibility and Status Dashboard",
+        "image_url": "images/impact_visibility_status_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "On-Call Management",
+        "image_url": "images/on_call_management.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Alerting",
+      "Category::Collaboration",
+      "Category::Incidents",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Offering::UI Extension",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
Please also specify the types of data you will be sending into Datadog and pulling out of Datadog, i.e.:

`Submitted Data Type::Metrics`
`Queried Data Type::Events`